src/components/CustomWearablePreview/CustomWearablePreview.tsx

```diff
@@ -17,17 +17,17 @@ export const CustomWearablePreview = (props: Props) => {
   useEffect(() => setIsLoading(true), [props.profile])
   const isUnityWearablePreviewEnabled = flags[FeatureFlagsKeys.UNITY_WEARABLE_PREVIEW]
   const [isLoadingUserAgentData, userAgentData] = useAdvancedUserAgentData()
-  const isIos = useMemo(
-    () => !isLoadingUserAgentData && (userAgentData?.mobile || userAgentData?.tablet) && userAgentData.os.name === 'iOS',
-    [isLoadingUserAgentData, userAgentData]
-  )
-  const isSafari = useMemo(
-    () => !isLoadingUserAgentData && userAgentData?.browser.name === 'Safari',
-    [isLoadingUserAgentData, userAgentData]
-  )
+  const isUnityWearablePreviewAllowed = useMemo(() => {
+    if (isLoadingUserAgentData) return false
+
+    const isIos = (userAgentData?.mobile || userAgentData?.tablet) && userAgentData.os.name === 'iOS'
```

Comment:
Isn't just enough with checking that the user is visiting the site from iOS?
```suggestion
    const isIos = userAgentData.os.name === 'iOS'
```